.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - "*"
+
+permissions:
+  contents: write
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Build bcc
```

Comment:
when we use deb or rpm, what will happen if bcc is not installed in a host?

Reply:
KubeArmor will fail to start as of now..